src/crate/client/sqlalchemy/tests/create_table_test.py

```diff
@@ -181,6 +181,51 @@ class DummyTable(self.Base):
         self.Base.metadata.create_all()
         fake_cursor.execute.assert_called_with(
             ('\nCREATE TABLE t (\n\t'
-             'pk STRING, \n\t'
+             'pk STRING NOT NULL, \n\t'
              'tags ARRAY(OBJECT), \n\t'
              'PRIMARY KEY (pk)\n)\n\n'), ())
+
+    def test_table_with_nullable(self):
```

Comment:
```suggestion
    def test_column_nullable(self):
```

Reply:
#### Naming things

I know the naming of the existing test case functions is not consistent at some spots, but let's get better and start using the `test_column_` prefix for all test cases related to, well, _column_-specific tests - opposed to the other test cases functions, which mostly exercise table-level attributes.

With this new convention, I think it will be fine to drop the `_with_` infix.

Reply:
Changed all tests in this class to be named wither `test_table_*` or `test_column_*`.